index.html

```diff
@@ -309,6 +309,25 @@ <h2>Skills</h2>
 					</ul>
 				</div>
 			</section>
+			
+			<section class="contact_section">
+				<div class="contact_section_container">
+					<h2>Contact me</h2>		
+					<div class="contact_text">
+						<p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+					</div>
+					<form autocomplete="on">						
```

Comment:
I suggest you add formspree to your contact form. As per the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) You should "Collect data by using the [Formspree service](https://formspree.io/html)."
<img width="1679" alt="Screenshot 2022-11-03 at 12 35 01" src="https://user-images.githubusercontent.com/48546992/199688025-cd958d3a-3893-4219-989c-37fddb1409c3.png">